sdk/olm_api_client/v1/client.py

```
@@ -49,15 +52,15 @@ async def _stream_response(
                             except json.JSONDecodeError:
                                 continue
         except httpx.RequestError as e:
-            logger.error(f"Ollama API streaming request failed: {e}")
+            logger.error(f"Olm API v1 streaming request failed: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)